src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/TreeViewTests.cs

```diff
@@ -4711,7 +4711,7 @@ public static IEnumerable<object[]> TreeViewNodeSorter_TestData()
     [MemberData(nameof(TreeViewNodeSorter_TestData))]
     public void TreeViewNodeSorter_Set_GetReturnsExpected(IComparer value)
     {
-        using TreeView treeView = new()
+        using TreeView treeView = new ()
```

Comment:
Please remove the space between ‘new’ and ‘()’. We typically do not have a space between those when doing target type new in this repo.
```suggestion
        using TreeView treeView = new()
```